image_test.go

```diff
@@ -38,6 +38,23 @@ func TestImageFit(t *testing.T) {
 	}
 }
 
+func TestRotatedImageFit(t *testing.T) {
```

Comment:
Tests, awesome!

I think this test is a bit heavy-duty and too specific for testing if the orientation logic works well. This test also does a lot of other stuff, like testing if fetching the Exif works well. Imo that falls outside the responsibility here though. I think it's safe to leave this test in, but another one is needed to properly test the new orientation logic.

When the new orientation logic is extracted into a separate function, it's very easy to write a [table-test](https://github.com/golang/go/wiki/TableDrivenTests) to test for several, currently untested, cases.

Reply:
Ideally I would like to replace the current jpeg with a rotated (CW 90 or 270) one. But currently it would break other tests. I removed now this test and I can open an new issue, is this ok for you?